packages/core/smoke-test/smoke-test.js

```diff
@@ -88,23 +90,38 @@ const copyTestApps = workdir => {
   TEST_APPS.forEach(appName => {
     const srcAppDir = path.join(repoRoot, 'example-apps', appName);
     const destAppDir = path.join(workdir, appName);
-    fs.copySync(srcAppDir, destAppDir);
+    fs.copySync(srcAppDir, destAppDir, {
+      filter: (src, dest) => !src.includes('node_modules/')
+    });
   });
 };
 
-const npmInstalls = (packagePath, workdir) => {
-  spawnSync('npm', ['install'], {
-    encoding: 'utf8',
-    cwd: workdir
-  });
-  spawnSync('npm', ['install', 'zapier-platform-cli'], {
-    encoding: 'utf8',
-    cwd: workdir
-  });
-  spawnSync('npm', ['install', '--no-save', packagePath], {
-    encoding: 'utf8',
-    cwd: workdir
-  });
+const npmInstalls = (coreZipPath, workdir) => {
+  // When releasing a new core version, example apps would have bumped their
+  // core before the new core version is published to npm. So we need to patch
+  // example app's package.json to use local core temporarily to avoid `npm
+  // install` error.
+  const packageJsonPath = path.join(workdir, 'package.json');
+  const origPackageJsonText = fs.readFileSync(packageJsonPath);
+  const packageJson = JSON.parse(origPackageJsonText);
+  packageJson.dependencies[CORE_PACKAGE_NAME] = `file:${coreZipPath}`;
```

Comment:
this might run afoul of #14 but if it works, then that's great!

Reply:
Thanks for the callout! `coreZipPath` is an absolute path so it works here.